packages/dara-components/dara/components/graphs/components/causal_graph_viewer.py

```diff
@@ -67,7 +83,7 @@ class CausalGraphViewer(BaseGraphComponent):
         offer any protection against edge crossings.
 
     ```python
-    from dara.components.graphs import CausalGraphViewer
+    from dara.components import CausalGraphViewer
     from dara.components.graphs.graph_layout import PlanarLayout
```

Comment:
You can also import the layouts from top level :)

Reply:
great call! done